docker/release/config/.vscode/c_cpp_properties.json

```diff
@@ -5,7 +5,7 @@
             "includePath": [
                 "${workspaceFolder}/**",
                 "${CUDA_QUANTUM_PATH}/**",
-                "/opt/llvm/lib/clang/**",
+                "/usr/local/llvm/lib/clang/**",
```

Comment:
Will installing to `/usr/local` clash with any of the default packing tools? e.g., if we do `apt-get install foo`, that might install to `/usr/local`?

Reply:
I don't think so for the packages we have.

Reply:
I don't. I tend to stick to ubuntu and ignore the rest.